addons/event/models/event_event.py

```diff
@@ -176,6 +176,20 @@ def _default_event_mail_ids(self):
         'Start sale date', compute='_compute_start_sale_date',
         help='If ticketing is used, contains the earliest starting sale date of tickets.')
 
+    def name_get(self):
```

Comment:
Better avoid defining name_get in the middle of fields definition, this breaks the file structure. It is generally defined with CRUD (create / read / write / unlink + name_get, ...) methods (global file ordering is defaults, fields, compute, constraints, CRUD, actions, ...)